packages/playground/src/pages/Helpers/Helpers.js

```diff
@@ -25,11 +25,14 @@ getPaymentMethods({ amount, shopperLocale }).then(async paymentMethodsResponse =
     // Adyen Giving
     window.donation = checkout
         .create('donation', {
-            onDonate: (state, component) => console.log({ state, component }),
+            onDonate: (state, component) => {
+                console.log({ state, component });
+                setTimeout(() => component.setStatus('ready'), 1000);
```

Comment:
Why do we need the setTimeout here?

Reply:
Just to 'simulate' a payment in the playground, otherwise the Donation component will be in 'loading' state forever once we click in the 'Donate' button